src/Send/MemoField.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
```

Comment:
for the file name may be just `Memo`, it would be nice if it represents more of the use case over the implementation